introduction.md

```diff
@@ -3,3 +3,8 @@
 My name is Spencer Kingman and I am happy to be joining the team.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?